backend/Application/Api/GraphQL/Search/SearchResult.cs

```diff
@@ -15,7 +15,7 @@ namespace Application.Api.GraphQL.Search;
 public class SearchResult
 {
     private static readonly Regex HashRegex = new("^[a-fA-F0-9]{1,64}$");
-    private static readonly Regex AccountAddressRegex = new("^[a-zA-Z0-9]{1,64}$");
+    private static readonly Regex Base58Regex = new("^[1-9A-HJ-NP-Za-km-z]{1,64}$");
```

Comment:
**Suggestion:** This regex could be more precise if split into two.

- An account address in base58 will always exactly 50 characters.
- A token address in base58 encodes the token ID as part of it, and the token ID is allowed to be up to 256 bytes, meaning the token address is variable length and can become fairly large (way more than 64, not sure the exact number).

Reply:
Fix in PR https://github.com/Concordium/concordium-scan/pull/185